.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,45 @@
+name: Continuous Integration
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
```

Comment:
So at the moment, we have an AzDO job that runs every time a push to a PR is made, and then once every 24 hours on master...

Based on that, this looks to fulfill the PR change, however I'm not sure that this would cover the 24-hour requirement. Also, it appears that it will re-run when pushed to master.

Can you confirm if this is the case? And if so, would we be able to change this to only PRs and a 24-hour schedule?
(If more information is wanted on schedules for workflows, you can look [here](https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#scheduled-events) 😄 )

Reply:
Done in 24ebb16